notebooks/data-sources/TestGrid/metrics/build_pass_failure.ipynb

```diff
@@ -2,25 +2,25 @@
  "cells": [
```

Comment:
<p>Do you still need to limit this number? Won't only looking at "overall" tests drop the number of rows to a more manageable number?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Yes, we can remove the limit here </p>